### PR TITLE
Make 404 great again

### PR DESCRIPTION
**Pull request by SonicHack2**
i am donald trump :Keepo:

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -20,10 +20,10 @@ <h2 class="subtitle">
 <div class="container">
   <div class="content" style="margin: 10px;">
     <p>
-	Make sure you typed the correct URL.<br>
+	Make sure you typed the correct URL, you spastic.<br>
 	If that fails, try <a href="/">going to the homepage</a> to find it.<br>
 	If that fails, then it probably doesn't exist here.
-	If navigating through the website fails, you're probably running Windows ME
+	If navigating through the website fails, jeff
```
Comment (TheRandomMelon) [inline 404.html]: i don't like jeff :(

Comment (SonicHack2) [reply 404.html]: i dont care >:)

File: 404.html [hunk 2]
```diff
@@ -23,7 +23,7 @@ <h2 class="subtitle">
 	Make sure you typed the correct URL, you spastic.<br>
 	If that fails, try <a href="/">going to the homepage</a> to find it.<br>
 	If that fails, then it probably doesn't exist here.
-	If navigating through the website fails, jeff
+	If navigating through the website fails, https://feheroes.gamepedia.com/Veronica
```
Comment (TheRandomMelon) [inline 404.html]: fire emblem is gay :(

Comment (SonicHack2) [reply 404.html]: NO U!!!!

Comment (SonicHack2) [reply 404.html]: https://feheroes.gamepedia.com/Veronica#Portrait

File: 404.html [hunk 3]
```diff
@@ -20,10 +20,10 @@ <h2 class="subtitle">
 <div class="container">
   <div class="content" style="margin: 10px;">
     <p>
-	Make sure you typed the correct URL.<br>
+	Make sure you typed the correct URL, you spastic.<br>
 	If that fails, try <a href="/">going to the homepage</a> to find it.<br>
 	If that fails, then it probably doesn't exist here.
-	If navigating through the website fails, you're probably running Windows ME
+	If navigating through the website fails,
```
Comment (TheRandomMelon) [inline 404.html]: 10

**Pull request merged**